src/core/frame.c

```diff
@@ -1227,13 +1245,15 @@ QuicAckFrequencyFrameDecode(
     _Out_ QUIC_ACK_FREQUENCY_EX* Frame
     )
 {
+    QUIC_ACK_FREQUENCY_EXTRAS Extras;
     if (!QuicVarIntDecode(BufferLength, Buffer, Offset, &Frame->SequenceNumber) ||
         !QuicVarIntDecode(BufferLength, Buffer, Offset, &Frame->PacketTolerance) ||
         !QuicVarIntDecode(BufferLength, Buffer, Offset, &Frame->UpdateMaxAckDelay) ||
-        !QuicUint8tDecode(BufferLength, Buffer, Offset, &Frame->IgnoreOrder) ||
-        Frame->IgnoreOrder > 1) { // IgnoreOrder should only be 0 or 1.
+        !QuicUint8tDecode(BufferLength, Buffer, Offset, &Extras.Value)) {
         return FALSE;
     }
+    Frame->IgnoreOrder = Extras.IgnoreOrder;
```

Comment:
Should the `FrameType` value be validated here to ensure it's the expected value?

Reply:
In `QUIC_ACK_FREQUENCY_EXTRAS`, `FrameType` is 6 bits, and I thought that was a useful value. If it's just a singular bit, I guess it doesn't need validation. Though, maybe the remaining 5 bits need to be validated as zero?

Reply:
Oops. `FrameType` should be `Reserved` and set to zero on write and ignored on read. So we have no functional issue here, but a variable name issue.